test/functional/test-video-docking.js

```diff
@@ -90,18 +92,44 @@ describes.repeated('', {
       });
     }
 
-    function createSlotElement(rect, ratio = 0) {
+    function maybeCreateSlotElementLtwh(left, top, width, height, ratio = 0) {
+      if (!useSlot) {
+        return;
+      }
       const impl = createAmpElementMock('amp-layout');
       impl.element.id = slotId;
       impl.element.setAttribute('layout', 'fill');
-      stubLayoutBox(impl, rect, ratio);
+      stubLayoutBox(impl, layoutRectLtwh(left, top, width, height), ratio);
       env.win.document.body.appendChild(impl.element);
 
       querySelectorStub
           .withArgs('#' + slotId)
           .returns(impl.element);
+    }
+
+    function setValidAreaWidth() {
+      mockAreaWidth(400);
+    }
+
+    function mockInvalidAreaWidth() {
+      const min = 320;
+      mockAreaWidth(min - 1);
+    }
+
+    function mockAreaWidth(width) {
+      sandbox.stub(docking, 'getAreaWidth_').returns(width);
```

Comment:
Referencing variables from closures instead of passing them as arguments can make test code not explicit enough. Consider passing the `docking` variable as an argument to make the behavior of this call more obvious to future readers.

Reply:
In this case `docking` is a global set on `beforeEach`. Larger change, so coming later.